File: Angular/satva/src/app/customer.ts
import { Address } from "./address";

export class Customer { 
    private rewardPoints:number=0;
    public get rewardpoints(): number {
        return this.rewardPoints;
    }
    public set rewardpoints(value: number) {
        this.rewardPoints = value;
    }
   public firstName: string = "";
    public  get firstname(): string {
        return this.firstName;
    }
    public set firstname(value: string) {
        this.firstName = value;
    }
	private lastName: string = "";
    public get lastname(): string {
        return this.lastName;
    }
    public set lastname(value: string) {
        this.lastName = value;
    }
	private emailId: string = "";
    public get emailid(): string {
        return this.emailId;
    }
    public set emailid(value: string) {
        this.emailId = value;
    }
	private passWord: string = "";   
    public get password(): string {
        return this.passWord;
    }
    public set password(value: string) {
        this.passWord = value;
    }
    private address: Address = new Address();
    public get add(): Address {
        return this.address;
    }
    public set add(value: Address) {
        this.address = value;
    }
    
    
}

File: SpringBoot/demo/src/main/java/com/example/rps/beans/sell/Customer.java
package com.example.rps.beans.sell;

import com.example.rps.beans.Address;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;


@JsonIgnoreProperties(ignoreUnknown = true)
public class Customer {
	private int custId;
	private String firstName;
	private String lastName;
	private String emailId;
	private String passWord;
	private Address address;
	private int rewardPoints;
	
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public String getEmailId() {
		return emailId;
	}
	public void setEmailId(String emailId) {
		this.emailId = emailId;
	}
	public String getPassWord() {
		return passWord;
	}
	public void setPassWord(String passWord) {
		this.passWord = passWord;
	}
	public Address getAddress() {
		return address;
	}
	public void setAddress(Address address) {
		this.address = address;
	}
	public int getCustId() {
		return custId;
	}
	public void setCustId(int custId) {
		this.custId = custId;
	}
	public int getRewardPoints() {
		return rewardPoints;
	}
	public void setRewardPoints(int rewardPoints) {
		this.rewardPoints = rewardPoints;
	}
	
}
